Lib/tkinter/__init__.py

```diff
@@ -4365,13 +4369,14 @@ def __init__(self, master=None, cnf={}, **kw):
         STANDARD OPTIONS
 
             background, borderwidth, cursor, height,
-            orient, relief, width
+            orient, proxybackground, proxyborderwidth,
```

Comment:
height and width were misplaced as standard. They and the 3 new proxy... options should be moved down.